cfpb/sbl-frontend
# [BUG][LOGIN] Refresh with query string problem

Closes #25 

#### Changes
- [fix] Refreshing after logging in no longer provides an unlogged-in state

#### How to test this PR

1. Run `sh start.sh` to start the app with keycloak and api via docker-compose setup
2. Click on `Login` at upper-right
3. Create a login
4. Login

There should no longer be the querystring in the URL and refreshing should not show an unlogged-in state.
